geochronometria.csl

```diff
@@ -81,7 +81,7 @@
         </group>
       </if>
       <else>
-        <text variable="genre" suffix=". "/>
+        <text variable="genre"/>
         <group delimiter=": ">
           <text variable="publisher"/>
           <text variable="publisher-place"/>
```

Comment:
@rmzelle, sorry for the late reply. And you're right, in some cases this change could cause problems. I group entire block and upload new version in a moment, this should correct the error.